src/Cms/Item.php

```diff
@@ -109,7 +109,7 @@ public function parent()
      * Returns the sibling collection
      * This is required by the HasSiblings trait
      *
-     * @return \Kirby\Editor\Blocks
+     * @return self::ITEMS_CLASS
```

Comment:
I think this will break our automatic reference docs. I couldn't find any actual information whether this is a supported format for phpdocumentor - what insight have you found there?

Reply:
On line 37 we define `\Kirby\Cms\Items` as type of the `siblings` property. So maybe we should just hint that here?

Reply:
It's a protected method, so it's not going to be in the reference, is it? But in general you are right, the format is a bit non-standard. Psalm supports its own annotations that override the default `@return` annotation, so I now used that one so Psalm is happy.

The issue with just using `\Kirby\Cms\Items` would be that it wouldn't be correct for classes that extend the `Item` class. That's why we have the constant `ITEMS_CLASS` after all. For example the `Block` class sets `\Kirby\Cms\Blocks`. If we fixed it for the parent class, Psalm wouldn't know the right type for further checks of the children classes and their dependents.

If you agree with my solution with `@psalm-return`, please resolve this review comment. Otherwise please let me know.